scripts/konflux_resolve.py

```
@@ -0,0 +1,1209 @@
+                    "wheel_files": pypi_result["wheel_files"],
+                }
+
+            pinned_version = resolved[norm]["version"]
+            visit_key = f"{norm}=={pinned_version}"
+            if visit_key in visited_queue:
+                continue
+            visited_queue.add(visit_key)
+
+            try:
+                trans_deps = self.pypi.get_requires_dist(norm, pinned_version)
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)